src/utilities/cast.ts

```diff
@@ -1,21 +1,28 @@
-function arraysToFloat32Arrays(arrays) {
-  const result = [];
+export function arraysToFloat32Arrays(arrays: number[][]): Float32Array[] {
```

Comment:
Since we have updated export style. i.e. from default to named exports, I think we need to update the referred imports too.  (References for these exports).

Reply:
The files that use these utils are 

`recurrent/rnn-time-step.js`

```js
const {
  arraysToFloat32Arrays,
  arrayToFloat32Arrays,
  objectsToFloat32Arrays,
  objectToFloat32Arrays,
  objectToFloat32Array,
} = require('../utilities/cast');
```

`neural-network.js`:

```js
const { arrayToFloat32Array } = require('./utilities/cast');
```

Which are using the restructured imports, is that correct? If not, please advise

Reply:
Yes, this is correct. 👍